AEPCore/Sources/core/MobileCore.swift

```diff
@@ -94,13 +94,23 @@ public final class MobileCore: NSObject {
     ///   - responseCallback: Callback to be invoked with `event`'s response `Event`
     @objc(dispatch:responseCallback:)
     public static func dispatch(event: Event, responseCallback: @escaping (Event?) -> Void) {
-        EventHub.shared.registerResponseListener(triggerEvent: event, timeout: 1) { event in
+        EventHub.shared.registerResponseListener(triggerEvent: event, timeout: 2) { event in
```

Comment:
should we make this timeout configurable? or at least explain that it exists in the docs for the method?

Reply:
will address this in a separate pr